testdata/config-locations.properties

```diff
@@ -1 +1 @@
-config.location=${sys:user.home}/.motech/configtest/, ${sys:user.home}/.motech/config/, /etc/motech/
\ No newline at end of file
+config.location=${sys:user.home}/.motech/configtest, ${sys:user.home}/.motech
```

Comment:
should stay as it was, with ${sys:user.home} getting eventually replaced with ${motech.dir}